test/OpenSpaceToolkit/Astrodynamics/Access/VisibilityCriterion.test.cpp

```diff
@@ -437,13 +437,73 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_Access_VisibilityCriterion, AERMaskIsSatis
         const auto aerMask = visibilityCriterion.as<VisibilityCriterion::AERMask>();
         ASSERT_TRUE(aerMask.has_value());
 
-        AER aer(Angle::Degrees(22.5), Angle::Degrees(20.0), Length::Meters(500e3));
+        AER aer(Angle::Degrees(22.5), Angle::Degrees(12.6), Length::Meters(500e3));
 
         EXPECT_TRUE(aerMask.value().isSatisfied(aer));
 
-        AER aerOutside(Angle::Degrees(22.5), Angle::Degrees(5.0), Length::Meters(500e3));
+        AER aer2(Angle::Degrees(0.0), Angle::Degrees(10.1), Length::Meters(500e3));
```

Comment:
Should we add a case where the azimuth beyond 360.0 to test the wrap around?

Reply:
Done